website/docs/r/rds_db_node.html.markdown

```diff
@@ -0,0 +1,108 @@
+---
+subcategory: "RDS"
+layout: "alicloud"
+page_title: "Alicloud: alicloud_rds_db_node"
+sidebar_current: "docs-alicloud-resource-rds-db-node"
+description: |-
+  Provide RDS cluster instance to increase node resources.
+---
+
+# alicloud\_rds\_db\_node
+
+Provide RDS cluster instance to increase node resources.
+-> **NOTE:** Available in 1.202.0+.
+
+## Example Usage
+
+```
+variable "name" {
+  default = "tf-testaccrdsdbnodes"
+}
+
+data "alicloud_db_zones" "default" {
+  engine                   = "MySQL"
+  engine_version           = "8.0"
+  instance_charge_type     = "PostPaid"
+  category                 = "cluster"
+  db_instance_storage_type = "cloud_essd"
+}
+
+data "alicloud_db_instance_classes" "default" {
+  zone_id                  = data.alicloud_db_zones.default.zones.0.id
+  engine                   = "MySQL"
+  engine_version           = "8.0"
+  category                 = "cluster"
+  db_instance_storage_type = "cloud_essd"
+  instance_charge_type     = "PostPaid"
+}
+
+data "alicloud_vpcs" "default" {
+	name_regex = "^default-NODELETING$"
+}
+data "alicloud_vswitches" "default" {
+	vpc_id 		 = data.alicloud_vpcs.default.ids.0
+	zone_id      = data.alicloud_db_zones.default.ids.0
+}
+
+resource "alicloud_vswitch" "vswitch" {
+  count             = length(data.alicloud_vswitches.default.ids) > 0 ? 0 : 1
+  vpc_id            = data.alicloud_vpcs.default.ids.0
+  cidr_block        = cidrsubnet(data.alicloud_vpcs.default.vpcs[0].cidr_block, 8, 8)
+  zone_id           = data.alicloud_db_zones.default.ids.0
+  vswitch_name      = var.name
+}
+
+locals {
+  vswitch_id = length(data.alicloud_vswitches.default.ids) > 0 ? data.alicloud_vswitches.default.ids[0] : concat(alicloud_vswitch.vswitch.*.id, [""])[0]
+}
+
+resource "alicloud_db_instance" "default" {
+  engine                   = "MySQL"
+  engine_version           = "8.0"
+  db_instance_storage_type = "cloud_essd"
+  instance_type            = data.alicloud_db_instance_classes.default.instance_classes.0.instance_class
+  instance_storage         = data.alicloud_db_instance_classes.default.instance_classes.0.storage_range.min
+  vswitch_id               = local.vswitch_id
+  instance_name            = var.name
+  zone_id 				   = data.alicloud_db_zones.default.ids.0
+  zone_id_slave_a          = data.alicloud_db_zones.default.ids.0
+}
+
+resource "alicloud_rds_db_node" "node" {
+  db_instance_id = alicloud_db_instance.default.id
+  class_code     = alicloud_db_instance.default.instance_type
+  zone_id        = alicloud_db_instance.default.zone_id
+}
+```
+
+## Argument Reference
+
+The following arguments are supported:
+
+* `db_instance_id` - (Required, ForceNew) The Id of instance that can run database.
+* `class_code` - (Required, ForceNew) The specification information of the node.
+* `zone_id` - (Required, ForceNew) The zone ID of the node.
+
+## Attributes Reference
+
+The following attributes are exported:
+
+* `id` - The resource ID in terraform of node.
```

Comment:
The value formats as `<db_instance_id>:<node_id>`.

Reply:
done